datastax-java-driver/src/main/scala/com/avast/sst/datastax/DatastaxModule.scala

```diff
@@ -0,0 +1,176 @@
+package com.avast.sst.datastax
+
+import cats.effect.{Resource, Sync}
+import com.avast.sst.datastax.config.DatastaxDriverConfig
+import com.avast.sst.datastax.DatastaxHelper._
+import com.datastax.oss.driver.api.core.CqlSession
+import com.datastax.oss.driver.api.core.config.DefaultDriverOption._
+import com.datastax.oss.driver.api.core.config.{DriverConfigLoader, ProgrammaticDriverConfigLoaderBuilder => DriverBuilder}
+import javax.net.ssl.SSLContext
+
+object DatastaxModule {

```

Comment:
This should also be named differently, probably `DatastaxJavaDriverModule`.

Reply:
Renamed to `CassandraDatastaxDriver`